app/controllers/character_types_controller.rb

```diff
@@ -26,6 +26,7 @@ def create
       if @character_type.save
         format.html { redirect_to @character_type }
       else
+        flash.now[:alert] = "Character not saved! Please provide a unique name"
```

Comment:
@StefanGajic I was thinking something more in line with displaying all the errors that happened when you tried to perform a save (e.g. `@character_types.errors`) and to do it in the template, not just flash.